tests/perf/actor_reminder/actor_reminder_test.go

```diff
@@ -39,7 +39,7 @@ const (
 	appName         = "perf-actor-reminder-service"
 
 	// Target for the QPS
-	targetQPS = 56
+	targetQPS = 50
```

Comment:
Lowering this to acct for json being made the default, which isn't as performant as using protobufs for storage - the diff is quite noticeable with the target app CPU time. Using json requires the app to marshal and unmarshal the data  increasing the target app CPU time vs using protobufs which significantly lowers the target app CPU time.

This target app CPU time diff btw protobuf & json is: 
JSON: 7m CPU time 
Protobuf: 1m CPU time